gno.land/cmd/gnoland/main.go

```diff
@@ -71,6 +72,13 @@ func (c *gnolandCfg) RegisterFlags(fs *flag.FlagSet) {
 		"quit after initialization, don't start the node",
 	)
 
+	fs.BoolVar(
+		&c.unsafeResetAll,
+		"unsafe-reset-all",
```

Comment:
```suggestion
		"unsafe-reset-before-start",
```

Maybe better this way?

Not convinced if it's logical, despite my personal interest in using the flag.